microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1184,46 +1184,73 @@ Get information about running WebView2 processes, exiting processes, and failed
 
 * `CoreWebView2` Class:
    * [CoreWebView2.BrowserProcessId Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.browserprocessid)
+   * [CoreWebView2.Reload Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.reload)<!--todo: add?  listed in other section(s)-->
```

Comment:
@lflores-ms - pls find "todo" in this file and delete links or delete or trim the hidden comments.  I added links that are present in article "Handling process-related events in WebView2" but were not in this Process Management section of "Overview of APIs" article
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis#process-management

Reply:
resolved todo's